src/main/java/digital/slovensko/autogram/core/eforms/EFormUtils.java

```diff
@@ -220,23 +222,33 @@ public static String getNamespaceFromEformXml(Document xml) {
         return null;
     }
 
-    public static byte[] getResource(String url) {
+    public static byte[] getResource(String url) throws ServiceUnavailableException {
         var offlineFileLoader = new FileCacheDataLoader();
         offlineFileLoader.setCacheExpirationTime(21600000);  // 6 hours
         offlineFileLoader.setDataLoader(new CommonsDataLoader());
 
         DSSDocument xsltDoc;
```

Comment:
už keď to chytáme, zmeň, prosím, názov tohto na niečo všeobecnejšie. Xslt bola prvá vec, čo sa kedysi sťahovala, teraz je to už hocijaký doc.